docs/en-us/doc_flask.md

```diff
@@ -104,9 +106,108 @@ Here's what you will get:
 ![flask-0](https://github.com/chenjiandongx/pyecharts/blob/master/images/flask-0.gif)
 
 
-## Conclusion
+### Conclusion
 
 It is extremely simple to include a pyexcharts instance in a Flask app. Please go
 ahead and put your own charts in flask.
 
 For your reference, please find the example code in `pyecharts/document` folder.
+
+
+
+## Way II :Render Using Template Functions
```

Comment:
I think Way II is intended for developer who has already django/flask exposure. and the existing one I wrote is intended for people who little or no prior experience. so you may adjust the wording to guide the reader to the right section.

Reply:
Yeah,accept your idea.I think it is better to move these words to a new project.